pytorch_toolkit/formula_recognition/im2latex/models/backbones/original_harvard_bb.py

```diff
@@ -0,0 +1,47 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import torch.nn as nn
+
+
+class Im2LatexBackBone(nn.Module):
```

Comment:
do you need this?

Reply:
For some datasets this backbone gives better result then resnet-like